app/controllers/home_controller.rb

```diff
@@ -50,9 +50,9 @@ def first_login_help
 
   # home page itself
   def index
-    @user_count = User.count
-    @work_count = Work.posted.count
-    @fandom_count = Fandom.canonical.count
+    @user_count = Rails.cache.fetch("User_count", :expires_in => ArchiveConfig.HOMEPAGE_CACHE_TIME , :race_condition_ttl => 5 ){ User.count}
+    @work_count = Rails.cache.fetch("Work_count", :expires_in => ArchiveConfig.HOMEPAGE_CACHE_TIME , :race_condition_ttl => 5 ){ Work.posted.count }
+    @fandom_count = Rails.cache.fetch("Fandom_count", :expires_in => ArchiveConfig.HOMEPAGE_CACHE_TIME , :race_condition_ttl => 5 ){ Fandom.canonical.count }
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space missing to the left of {.<br>Space found before comma.<br>Space inside parentheses detected.